jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP_OBS

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalprepobs" -c "base ocnanal ocnanalprepobs"
```

Comment:
```suggestion
source "${HOMEgfs}/ush/jjob_header.sh" -e "prepoceanobs" -c "base prepoceanobs"
```
I don't think there is a need for the dependency to source `ocnanal` config file here at this stage.